meteor/server/api/integration/rabbitMQ.ts

```diff
@@ -248,7 +254,7 @@ class ChannelManager extends Manager<AMQP.ConfirmChannel> {
 				// Put the message back on the queue:
 				this.outgoingQueue.unshift(messageToSend)
 			} else {
-				logger.debug('RabbitMQ: message sent, waiting for ok...')
+				logger.debug('AMQP: message sent, waiting for ok...')
```

Comment:
@jstarpl FYI: This file should be renamed. It's called 'RabbitMQ', but it's using an 'AMQP' integration.
If you are the wrong person to tag, please let me know :)

Reply:
> AMQP is the core protocol for RabbitMQ